runtime/vm/rasdump.c

```diff
@@ -223,8 +223,14 @@ gpThreadDump(struct J9JavaVM *vm, struct J9VMThread *currentThread)
 		do {
 			if (currentThread->threadObject) {
 				j9object_t threadObject = currentThread->threadObject;
+#if JAVA_SPEC_VERSION >= 19
+				j9object_t threadHolder = J9VMJAVALANGTHREAD_HOLDER(currentThread, threadObject);
+				UDATA priority = J9VMJAVALANGTHREADFIELDHOLDER_PRIORITY(currentThread, threadHolder);
+				UDATA isDaemon = J9VMJAVALANGTHREADFIELDHOLDER_DAEMON(currentThread, threadHolder);
```

Comment:
This should check whether `NULL != threadHolder` as is done in `jvmtiRunAgentThread()`.